_layouts/docs.html

```diff
@@ -42,6 +41,13 @@
             <button class="button is-rounded" id="table-of-contents" onclick="tocToggler()">Table of contents</button>
             <div class="content">
                 <h1>{{page.title}}</h1>
```

Comment:
This looks good @gravitonian but I wonder if we can move the version number after the title to help improve search hits (as shown below)? 

```suggestion
                <h1>{{page.title}} {{page.version}}</h1>
```

It's also a really elegant solution as it updates the template, and saves us having to update every product landing page. However, we also need to be more flexible as some products don't display a version number (e.g. APA - on PaaS, all Mobile docs, Support Handbook):

- https://docs.alfresco.com/process-automation/latest/
- https://docs.alfresco.com/content-mobile/latest/

Can we exclude the version number if `versions` are not defined in `_config.yml`?

Reply:
Versions are specified in _config.yml, the above products (process-automation and content-mobile) displays version 1.0 as specified in _config.yml.

Reply:
Version is now in the product title